lib/cli/commands/serve.js

```diff
@@ -46,8 +46,12 @@ serve.builder = function(cli) {
 			"Listen to port 1337 and launch default browser with http://localhost:1337/test/QUnit.html");
 };
 
-serve.handler = function(argv) {
+serve.openUrl = async function(browserUrl) {
```

Comment:
Same here, adding a function just to make testing easier should be avoided if possible. What about [mock-require](https://github.com/boblauer/mock-require)?

The function is called just one time, so no real need to have it in a separate function.